### PR TITLE
Feature/sdl 0236 tire status mismatch

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Implements https://github.com/smartdevicelink/rpc_spec/issues/168

This PR is **[ready]** for review.

### Summary
Mark the parameters of the TireStatus structure as non-mandatory in the MOBILE_API

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by KhrystynaDubovyk** [comment]
@atiwari9 please review this PR

**Review by atiwari9 — APPROVED** [review]
Approved

**Comment by LitvinenkoIra** [comment]
@JackLivio This PR is ready for Livio review. Thank you!

**Review by atiwari9 — APPROVED** [review]

**Comment by LitvinenkoIra** [comment]
@theresalech This PR is ready for Livio review. Thank you!

File: MOBILE_API.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2021-04-14">
+<interface name="SmartDeviceLink RAPI" version="8.0.0" minVersion="1.0" date="2021-04-14">
```
Comment (Jack-Byrne) [inline MOBILE_API.xml]:
```suggestion
<interface name="SmartDeviceLink RAPI" version="8.0.0" minVersion="1.0" date="2021-10-27">
```

Tentatively set to prospected release date

**Comment by jordynmackool** [comment]
@LitvinenkoIra Please see Jacks' review feedback above and let us know if you have any questions. Thanks!

File: MOBILE_API.xml (hunk 1 above)
Comment (AKalinich-Luxoft) [reply MOBILE_API.xml]: @JackLivio fixed in https://github.com/smartdevicelink/rpc_spec/pull/296/commits/6f6829fcb03aa353174c6f066042e5b01d495a3c

**Comment by AKalinich-Luxoft** [comment]
> @LitvinenkoIra Please see Jacks' review feedback above and let us know if you have any questions. Thanks!

@jordynmackool all Livio comments have been processed

**Review by Jack-Byrne — APPROVED** [review]

**Pull request merged**